IBPSA/BoundaryConditions/WeatherData/BaseClasses/PartialConvertTime.mo

```diff
@@ -0,0 +1,58 @@
+within IBPSA.BoundaryConditions.WeatherData.BaseClasses;
+partial block PartialConvertTime
+  "Converts the simulation time to calendar time in scale of 1 year (365 days), or a multiple of a year"
+  extends Modelica.Blocks.Icons.Block;
+  parameter Modelica.Units.SI.Time weaDatStaTim(displayUnit="d") = 0
+    "Start time of weather data";
+  parameter Modelica.Units.SI.Time weaDatEndTim(displayUnit="d") = 31536000
+    "End time of weather data";
+  Modelica.Units.SI.Time simTim "Simulation time";
+  Modelica.Units.SI.Time calTimExp "Calendar time";
```

Comment:
@mwetter, thank you for the review and the comments. `simTim` is indeed `modTim` and I will change the description to "model Time", while with the "Exp" suffix I meant expression. Regarding the name of the variable itself, same as for `calTim`, those names cannot be used in the partial class, because they are already used in `IBPSA.BoundaryConditions.WeatherData.BaseClasses.ConvertTime` as the Real input `modTim` and the output `calTim`. In the partial class I did not add any Real input or output, because the names are different across the models where it is used (i.e. in `ConvertTime` the input is `modTim`, in `EquationOfTime` the input is `nDay`) at the same time `calTim` in other models, where the partial class is used, is an intermediate variable. I suggest changing `simTim` and `calTimExp` to `modTimAux` and `calTimAux` for Auxiliary or to `modTimCal` and `calTimCal` for calculated. Unless there is IBPSA code guideline for these cases.

Reply:
@EttoreZ : I see. Adding the suffice `Aux` as you proposed is good.

Reply:
I update the variable names adding `Aux` at the end of the variable names